avocado/core/loader.py

```diff
@@ -244,10 +244,13 @@ def load_test(self, test_factory):
                 sys.path.insert(0, test_module_dir)
                 f, p, d = imp.find_module(module_name, [test_module_dir])
                 test_module = imp.load_module(module_name, f, p, d)
-            except ImportError as details:
-                raise ImportError("Unable to import test's module with "
-                                  "sys.path=%s\n\n%s" % (", ".join(sys.path),
-                                                         details))
+            except:
+                # On load_module exception we fake the test class and pass
+                # the exc_info as parameter to be logged.
+                test_parameters['methodName'] = 'test'
```

Comment:
Now when I look at this, maybe we should use `<unknown>` or something like that, because it can be confusing when the code does not have `test` method....